app/src/angular/configs/app.run.js

```diff
@@ -36,13 +36,15 @@ function AppRun($rootScope, $log, $window, $timeout, $interval, $q, $state, $tra
     Wallet.SqlLiteService = SqlLiteService;
     Wallet.CommonService = CommonService;
     Wallet.EtherScanService = EtherScanService;
+    Wallet.LedgerService = LedgerService;
```

Comment:
Why are we monkeypatching these modules like this?

Reply:
This is a way to path some services into Token&Wallet classes, in order to access the services from the class. 
this is current, existing implementation if we need to change/refactor it, we should create another task/branch for this.
(this PR just stands for Ledger integration)